### ISSUE TITLE
How to convert syncbn to batchnormND?

### ISSUE THREAD

**Issue by Edwardmark** [pytorch/pytorch#48075]
I want to run a model with syncBn in cpu, so I have to convert syncBN to batchNormND, how can I  do that?
I just found a way to convert from bn to syncbn, but how to do the opposite? Thanks in advance.
[convert2syncbn](https://pytorch.org/docs/stable/generated/torch.nn.SyncBatchNorm.html?highlight=sync#torch.nn.SyncBatchNorm.convert_sync_batchnorm)

cc @albanD @mruberry

**Comment by gchanan** [comment]
I'm not sure if we have something that can do this automatically but I presume you can basically invert the implementation of sync_batch_norm: https://pytorch.org/docs/stable/_modules/torch/nn/modules/batchnorm.html#SyncBatchNorm.convert_sync_batchnorm

**Comment by mruberry** [comment]
This question is probably a better fit for our forums: https://discuss.pytorch.org/. Try asking there, too.

**Comment by Edwardmark** [comment]
https://discuss.pytorch.org/t/how-to-convert-syncbn-to-batchnormnd/103197 @mruberry I post a question in discuss, please help me out, thanks.
